### PR TITLE
Your new branch

### PR DESCRIPTION
**Pull request by snigdharao55**
https://github.com/snigdharao55/snigdharao55.github.io/blob/your-new-branch/pages/profiles/snigdharao.md

### CODE REVIEW

**Comment by snigdharao55** [comment]
https://github.com/snigdharao55/snigdharao55.github.io/blob/your-new-branch/pages/profiles/snigdharao.md

File: pages/profiles/snigdharao.md [hunk 1]
```diff
@@ -6,10 +6,19 @@
 ### BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
 ***
 As a recent graduate with a Master’s in Computer Science from California State University, Long Beach,I am passionate about coding and love to learn new tech stacks .I have  over 2 years of work experience and i love to travel
-# Projects
+---
```
Comment (Yurockkk) [inline pages/profiles/snigdharao.md]: @snigdharao55 By adding an underline here, it makes the whole paragraph above it into a H2 header. Is that what you want?

Comment (snigdharao55) [reply pages/profiles/snigdharao.md]: No , I will fix it now .

Comment (snigdharao55) [reply pages/profiles/snigdharao.md]: Yes , i have added h2(##) for it .Do i have to add any more tags?

**Comment by snigdharao55** [comment]
shall i remove h2 header?

File: pages/profiles/snigdharao.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+### Education
+* Master’s in Computer Science, California State University, Long Beach (GPA: 3.7) 
+*  Bachelor’s in Computer Science, Jawaharlal Nehru Technological University (GPA: 4.0)
+
+### Projects
+1. [TechFish Bank ](https://goo.gl/MmY1DK)
+    Technologies : Java, Spring Boot, Angular 2, jQuery
+2. [UberTruck]( https://goo.gl/N4xqxT) 
+    Technologies: Java, JSP, Servlets, JavaScript,AngularJs
+3. [Earthquake Visualization](https://goo.gl/TrbN7L.)
```
Comment (xyb994) [inline pages/profiles/snigdharao.md]: Is `.` part of the url?

File: pages/profiles/snigdharao.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Time zone : EDT, -0400
+OS : Mac osX
+BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
+
+As a recent graduate with a Master’s in Computer Science from California State University, Long Beach,I am passionate about coding and love to learn new tech stacks .I have  over 2 years of work experience and i love to travel
+
+### Education
+* Master’s in Computer Science, California State University, Long Beach (GPA: 3.7) 
+*  Bachelor’s in Computer Science, Jawaharlal Nehru Technological University (GPA: 4.0)
```
Comment (xyb994) [inline pages/profiles/snigdharao.md]: Would it be better to remove the extra space before `Bachelor’s in` to make it consistent with the previous line? This does not affect what it looks like in MDwiki, but to makes the raw document neat...

File: pages/profiles/snigdharao.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+ # **Snigdha Rao Madhavaram** 
+<p>
+ <img height="250" width="250" src="https://s3.amazonaws.com/mylifecylebuckhyd12/IMG_2569+(1).jpg">
+</p>
+
+Time zone : EDT, -0400
+OS : Mac osX
+BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
```
Comment (xyb994) [inline pages/profiles/snigdharao.md]: Would it be better to add a space after `BuzzWords :` to make it consistent with previous 2 lines?

File: pages/profiles/snigdharao.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+As a recent graduate with a Master’s in Computer Science from California State University, Long Beach,I am passionate about coding and love to learn new tech stacks .I have  over 2 years of work experience and i love to travel
+
+### Education
+* Master’s in Computer Science, California State University, Long Beach (GPA: 3.7) 
+*  Bachelor’s in Computer Science, Jawaharlal Nehru Technological University (GPA: 4.0)
+
+### Projects
+1. [TechFish Bank ](https://goo.gl/MmY1DK)
+    Technologies : Java, Spring Boot, Angular 2, jQuery
+2. [UberTruck]( https://goo.gl/N4xqxT)
```
Comment (xyb994) [inline pages/profiles/snigdharao.md]: There's an extra space before `https://goo.gl/N4xqxT`, it's not consistent with the formatting of other 2 items in the list, would it be better to remove it?

**Comment by bsocial** [comment]
Looks really good. +1

**Pull request merged**